internal/db/db.go

```diff
@@ -221,7 +228,7 @@ func (db *dB) GetComposes(orgId string, since time.Duration, limit, offset int)
 	}
 
 	var count int
-	err = conn.QueryRow(ctx, sqlCountActiveComposesSince, orgId, since).Scan(&count)
+	err = conn.QueryRow(ctx, sqlCountActiveComposesSince, orgId, since, pq.Array(IgnoreImageTypes)).Scan(&count)
```

Comment:
I don't see why the `pq.Array` cast is necessary, it compiles and runs fine without it.